gallery/src/styles/NxCard/NxCardVerticalLayoutExample.tsx

```diff
@@ -0,0 +1,266 @@
+            <a href="#" className="nx-text-link">Film sprawl wristwatch math-engine</a>
+          </footer>
+        </section>
+      </div>
+
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card" aria-label="card 7 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            42
+          </div>
+        </section>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card" aria-label="card 8 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__call-out">
+            <NxFontAwesomeIcon icon={faShapes} className="nx-card__call-out-icon" />
+          </div>
+          <footer className="nx-card__actions">
+            <a href="#" className="nx-text-link">Link</a>
+          </footer>
+        </section>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card nx-card--align-left gallery-card--66wide" aria-label="card 9 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">A wide card with left aligned content</h3>
+          </header>
+          <div className="nx-card__content nx-card__content--2-column">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faChartLine} className="nx-card__call-out-icon" />
+            </div>
+            <div className="nx-card__text">
+              Lots of content to force the card to a wider size sentient noodles sprawl tower corrupted cartel
+              carbon decay sentient post- singularity
+            </div>
+          </div>
+        </section>
+        <section className="nx-card" aria-label="card 1 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3">Card header</h3>
+          </header>
+          <div className="nx-card__content nx-card__content--2-column">
+            <div className="nx-card__call-out">
+              <NxFontAwesomeIcon icon={faShapes} className="nx-card__call-out-icon" />
+            </div>
+            <div className="nx-card__text">
+              Sentient post-singularity
+            </div>
+          </div>
+        </section>
+      </div>
+      <div className="nx-card-container nx-card-container--row">
+        <section className="nx-card gallery-card--49wide" aria-label="card 10 in row">
+          <header className="nx-card__header">
+            <h3 className="nx-h3 nx-h3--card-header">A card that's ~50% of available width</h3>
+          </header>
+          <div className="nx-card__content">
+            <div className="nx-card__text">
+              Lots of content to force the card to a wider size alcohol realism tower cardboard network sunglasses
```

Comment:
"Lots of content to force the card to a wider size" isn't true here anymore

Reply:
> "Lots of content to force the card to a wider size" isn't true here anymore

As above: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/38859601e2652ef2c8fd2d2d283de7d9654cafb2